t/admin/schema-validate.t

```diff
@@ -398,3 +398,44 @@ location /t {
 --- error_code: 400
 --- response
 {"error_msg":"allOf 1 failed: value should match only one schema, but matches none"}
+
+
+
+=== TEST 13: Check node_schema optional port
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            headers = {}
+            headers["X-API-KEY"] = "edd1c9f034335f136f87ad84b625c8f1"
+            local code, body = t('/apisix/admin/routes/1',
+                ngx.HTTP_PUT,
+                {
+                    uri = "/ip",
+                    upstream = {
+                        type = "roundrobin",
+                        nodes = {
+                            { host = "httpbin.org", weight = 1,}
+                        }
+                    },
+                    methods = {"GET"},
+                }, nil, headers
+            )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+
+
+
+=== TEST 14: Test route upstream
+--- request
+GET /ip
```

Comment:
@Revolyssup Would you like to update these test cases? it is better not to use `httpbin.org`, it is unstable, and we could check the response body like the other test cases, and no need to add `X-API-KEY` in the request header and so on

Reply:
okay

Reply:
> okay

I remove some useless code.